api/namex/resources/requests.py

```diff
@@ -382,9 +382,100 @@ def post(self, *args, **kwargs):
         return jsonify({'message': 'Not Implemented'}), 501
 
 # For sbc-auth - My Business Registry page.
-@cors_preflight("POST")
-@api.route('/search', methods=['POST'])
+
+
+@cors_preflight("GET, POST")
+@api.route('/search', methods=['GET', 'POST', 'OPTIONS'])
 class RequestSearch(Resource):
+    """Search for NR's."""
+
+    @staticmethod
+    @cors.crossdomain(origin='*')
+    # @jwt.requires_auth
```

Comment:
can we remove this line altogether?

Reply:
we need this. i commented this for testing